app/templates/components/learnergroup-members.hbs

```diff
@@ -1,18 +1,59 @@
 {{#liquid-if saving class='crossFade'}}
   {{wait-saving}}
 {{/liquid-if}}
-{{#each proxiedMembers as |userProxy|}}
-  <div class="form-col-12">
-    <label class="form-label learnergroup-username">{{userProxy.fullName}}</label>
-    <div class="form-data learnergroup-group-membership">
-      {{inplace-select
-        value=userProxy.lowestGroupInTree.id
-        options=learnerGroupOptions
-        save='changeLearnerGroup'
-        condition=userProxy.id
-        displayValueOverride=userProxy.groupDisplayValueString
-        selectPromptTranslation='learnerGroups.selectGroup'
-      }}
+
+{{#if multiEditModeOn}}
+  <div class="multi-edit__container">
+    <div class="multi-edit__box">
+      <label>{{t "learnerGroups.includeAll"}}</label>
+      {{input type="checkbox" checked=includeAll}}
     </div>
   </div>
-{{/each}}
+{{/if}}
+
+<div class="learnergroup-list">
+  {{#each proxiedMembers as |userProxy|}}
+    <div class="form-col-12">
+      <label class="form-label learnergroup-username">{{userProxy.fullName}}</label>
+      <div class="form-data learnergroup-group-membership">
+        {{inplace-select
```

Comment:
I think these can just be HTML <select> tags with actions.  Take a look at the inplaceSelect handlebars for an example.  Any reason not to do it that way?